src/main/window-manager.ts

```diff
@@ -175,18 +181,30 @@ export class WindowManager extends Singleton {
     return this.mainWindow;
   }
 
-  sendToView({ channel, frameInfo, data = [] }: { channel: string, frameInfo?: ClusterFrameInfo, data?: any[] }) {
+  private sendToView({ channel, frameInfo, data = [] }: SendToViewArgs) {
```

Comment:
This isn't used outside of this class so it really should be private, especially since the other helper functions cover all the cases.